### ISSUE TITLE
ActionController::UnknownFormat error from ChangeRequestsController

### ISSUE THREAD

**Issue by jvendetti** [ncbo/bioportal_web_ui#361]
From @alexskr:

> im seeing suspicious traffic hitting /change_requests on the UI.   These hits don't look legitimate since they return the 406 status code.

Apache logs:

```
47.76.209.138 - - [20/Nov/2024:22:32:28 -0800] "HEAD /change_requests/remove_synonym?concept_id=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0000118&concept_label=lung%2Bbud&concept_synonyms%5B%5D=lung%2Bbud&ont_acronym=UBERON HTTP/1.1" 406 - "-" "Mozilla/5.0 (Windows NT 6.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.2775.158 Safari/537.36"
47.76.209.138 - - [20/Nov/2024:22:32:28 -0800] "GET /change_requests/remove_synonym?concept_id=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0000118&concept_label=lung%2Bbud&concept_synonyms%5B%5D=lung%2Bbud&ont_acronym=UBERON HTTP/1.1" 406 - "-" "Mozilla/5.0 (Windows NT 6.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.2775.158 Safari/537.36"
47.76.99.127 - - [20/Nov/2024:22:32:32 -0800] "HEAD /change_requests/remove_synonym?concept_id=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0017145&concept_label=stem%2Bcell%2Bdivision&concept_synonyms%5B%5D=stem%2Bcell%2Brenewal&ont_acronym=UBERON HTTP/1.1" 406 - "-" "Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3501.80 Safari/537.36"
47.76.99.127 - - [20/Nov/2024:22:32:32 -0800] "GET /change_requests/remove_synonym?concept_id=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0017145&concept_label=stem%2Bcell%2Bdivision&concept_synonyms%5B%5D=stem%2Bcell%2Brenewal&ont_acronym=UBERON HTTP/1.1" 406 - "-" "Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3501.80 Safari/537.36"
```

UI logs:

```
F, [2024-11-20T22:50:04.484217 #59068] FATAL -- : [aaacc637-7fcd-4c7d-bc78-043b76d601f8]
[aaacc637-7fcd-4c7d-bc78-043b76d601f8] ActionController::UnknownFormat (ActionController::UnknownFormat):
[aaacc637-7fcd-4c7d-bc78-043b76d601f8]
[aaacc637-7fcd-4c7d-bc78-043b76d601f8] app/controllers/change_requests_controller.rb:59:in `require_login'
```

NewRelic logs:

```
/srv/ontoportal/bioportal_web_ui/shared/bundle/ruby/3.0.0/gems/actionpack-7.0.8/lib/action_controller/metal/mime_responds.rb:216:in `respond_to'
/srv/ontoportal/bioportal_web_ui/releases/20241121040948/app/controllers/change_requests_controller.rb:59:in `require_login'
```

**Comment by alexskr** [comment]
ActionController::UnknownFormat error is the most frequent error we have 
<img width="1066" alt="image" src="https://github.com/user-attachments/assets/5760ba6a-1372-44f5-bd60-040cb78a5d17">

**Comment by jvendetti** [comment]
Fixed by this commit: 29e7b2288ce02649e210bd38e9f82ca1092adf7d
